src/components/AddButton.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div class="grid place-items-end pt-16 pr-8">
+        <button class="add-button w-2/12 p-3 bg-button rounded-full text-white text-center" @click="addIndustry">
```

Comment:
Looks like this is a general component. But we are explicitly defining addIndustry and there are data related to Industry here. Pass the props and emit onclick event, and catch the onClick event on parent component.